src/resolvers/Organization/image.ts

```diff
@@ -0,0 +1,9 @@
+import { BASE_URL } from "../../constants";
+import { OrganizationResolvers } from "../../types/generatedGraphQLTypes";
+
```

Comment:
Add a comment here explaining why this was needed so people looking at this won't be confused.